spec/features/user_views_index_page_spec.rb

```diff
@@ -7,7 +7,21 @@
 # Links should lead to item show pages
 
 feature "User views index page to see items" do
+  let(:user) do
+    User.create(
+      email: "asdf@asdf.com",
+      password: "asdf1234"
+    )
+  end
+
   before(:each) do
+    user
+    visit root_path
+    click_link "Log in"
+    fill_in 'Email', with: "asdf@asdf.com"
+    fill_in 'Password', with: "asdf1234"
+    click_button "Log in"
+    
     @item1 = create(:item)
```

Comment:
You shouldn't be creating items in `before(:each)` and assigning them to instance variables.. It's preferred to use `let` statements. You can either use `let` to create the object whenever the variable name is referred to, or you can use `let!` to ensure that it's created before any scenario whether you refer to it by variable name or not